openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -140,8 +27,7 @@ class CourseWaffleFlag(LegacyWaffleFlag):
 
     Usage:
 
-       WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='my_namespace')
-       SOME_COURSE_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'some_course_feature', __name__)
+       SOME_COURSE_FLAG = CourseWaffleFlag('my_namespace', 'some_course_feature', __name__)
```

Comment:
@regisb: FYI: I removed the use of the namespace class in these usage instructions.